tom_tns/tns_api.py

```diff
@@ -34,6 +34,16 @@ def map_filter_to_tns(filter):
         return settings.BROKERS.get('TNS', {}).get('filter_mapping', {}).get(filter)
 
 
+def map_instrument_to_tns(instrument):
+    """ Checks if a instrument mapping was set in the settings, and if so returns the
+        mapped value for the instrument passed in
+    """
+    if submit_through_hermes():
+        return settings.DATA_SHARING.get('hermes', {}).get('INSTRUMENT_MAPPING', {}).get(instrument)
```

Comment:
Would it be difficult to make these the same capitalization? Is there a reason we don't?

Reply:
So in the initial tom_tns code it was uncapitalized there following the convention of the BROKERS section for TOM settings, but in my update it is capitalized in the DATA_SHARING section which follows the convention there. I believe the DATA_SHARING section is following the convention that other django settings use, so I'm inclined to think the BROKERS section is doing it unconventionally. But that's not really up to me to switch, its a tom_base decision.

Reply:
Brokers will be going away in 3.0 so we'll try to address that then.